gui/upgrade.cc

```diff
@@ -81,86 +75,10 @@ bool UpgradeCheck::isTestMode()
   return testing;
 }
 
-// In Qt 5.4, QSysInfo got classes that do this better. Travis is stuck
-// on older version of Qt, so keep it building there for now.
+// In Qt 5.4, QSysInfo got classes that do this better.
```

Comment:
Are we OK with the check in gui/main.cc handling the test for QT_VERSION_CHECK?  Maybe we move it into one of the .h's that's included everywhere so it'll fail earlier.


I'm OK with the new way being the only way and being free of that OS-specific clutter that we had.  It's always been a maintenance pain for us.

Reply:
I just took the version check out of upgrade.cc and fixed up the comment.  We still have gui/main.cc enforcing 5.7 minimum.